traveler_app/src/main/java/com/guestlogix/traveler/activities/SettingsActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.guestlogix.traveler.activities;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.guestlogix.traveler.R;
+import com.guestlogix.travelercorekit.utilities.TravelerPrefs;
+
+import static com.guestlogix.travelercorekit.utilities.UrlHelper.isValid;
+
+public class SettingsActivity extends AppCompatActivity {
+
+    private TravelerPrefs travelerPrefs = TravelerPrefs.getInstance(this);
+
+    private TextView sdkEndpointEditText;
+    private TextView authEndpointEditText;
+    private Button updateSDKEndpointButton;
+    private Button resetSDKEndpointButton;
+    private Button updateAuthEndpointButton;
+    private Button resetAuthEndpointButton;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_settings);
+
+        sdkEndpointEditText = findViewById(R.id.newSDkEndpointEditText);
+        authEndpointEditText = findViewById(R.id.newAuthEndpointEditText);
+        updateSDKEndpointButton = findViewById(R.id.updateSDkEndpointButton);
+        resetSDKEndpointButton = findViewById(R.id.resetSDkEndpointButton);
+        updateAuthEndpointButton = findViewById(R.id.updateAuthEndpointButton);
+        resetAuthEndpointButton = findViewById(R.id.resetAuthEndpointButton);
+
```

Comment:
Would it be possible to populate the edittexts with the current URL values? it would be super useful
![image](https://user-images.githubusercontent.com/7252786/65460996-19d22400-de21-11e9-82a4-a9922c02cc67.png)

Reply:
I've modified it to show whatever is current store in shared prefs